src/index.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+var _ = { extend: require('lodash.merge') };
+var tracker = require('tracker');
+
+var rememberMe = module.exports = {
+  key: 'rememberMe'
+};
+
+rememberMe.set = function( params ) {
+  var defaults = {
+    url: location.href
+  };
+  var data = _.extend( defaults, params );
+  try {
+    localStorage[rememberMe.key] = JSON.stringify( data );
```

Comment:
Can we use an existing module like this one here instead: https://www.npmjs.com/package/putainde-localstorage